pkg/endpoint/bpf.go

```diff
@@ -63,19 +64,21 @@ func (e *Endpoint) writeL4Map(fw *bufio.Writer, owner Owner, m policy.L4PolicyMa
 		}
 
 		redirect = common.Swab16(redirect)
-		entry := fmt.Sprintf("{%d,%d,%d}", dport, redirect, protoNum)
+		entry := fmt.Sprintf("%d,%d,%d,%d", index, dport, redirect, protoNum)
 		if array != "" {
 			array = array + "," + entry
 		} else {
 			array = entry
 		}
 
+		index += 1
```

Comment:
Is `i` here guaranteed to iterate from 0,1,2...,n? I thought (with my limited go knowledge) that range was allowed to iterate through a map in any arbitrary order.

Also note I'll need to do another push with the new macros shortly.